src/mapping/RadialBasisFctMapping.hpp

```diff
@@ -3,6 +3,7 @@
 #include <Eigen/Core>
 #include <Eigen/QR>
 
+#include "RadialBasisFctSolver.h"
```

Comment:
```suggestion
#include "mapping/RadialBasisFctSolver.hpp"
```

This will simplify the migration to full prefixes.